catkin_ws/src/controls/src/servers/state_quaternion_pid_server.py

```diff
@@ -65,10 +69,17 @@ def callback(self, goal):
 
```

Comment:
why sleep? instead of just letting the loop run?

Reply:
I wouldn't let me comment on the actual line, it's the one above

Reply:
we can let it run if you want